### PR TITLE
data:rezarex

### PR DESCRIPTION
**Pull request by rezarex**

### CODE REVIEW

**Review by SurajPratap10 — DISMISSED** [review]
LGTM

File: data/rezarex.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/wekessah",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "dev.to",
+      "url": "https://twitter.com/wekessah",
+      "icon": "fa-dev-to "
```
Comment (imbeshat) [inline data/rezarex.json:14]:
```suggestion
      "icon": "FaDev"
```

File: data/rezarex.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "bio": "Backend Engineer | Data Science | Python | Javascript | Always Learning Something New ",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/wekessah",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "dev.to",
+      "url": "https://twitter.com/wekessah",
```
Comment (imbeshat) [inline data/rezarex.json:13]: URL of dev.to

File: data/rezarex.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "fa-dev-to "
+    },
+      {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/marvin-wekesa-58112b44/",
+      "icon": "FaLinkedIn"
+    },
+     {
+      "name": "GitHub",
+      "url": "https://www.instagram.com/rezarex/",
```
Comment (imbeshat) [inline data/rezarex.json:23]: URL of GitHub

**Review by imbeshat — CHANGES_REQUESTED** [review]
Please, update the icons, and also update the links of dev.to and GitHub.

File: data/rezarex.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    },
+     {
+      "name": "dev.to",
+      "url": "https://twitter.com/wekessah",
+      "icon": "fa-dev-to "
+    },
+      {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/marvin-wekesa-58112b44/",
+      "icon": "FaLinkedIn"
```
Comment (imbeshat) [inline data/rezarex.json:19]:
```suggestion
      "icon": "FaLinkedin"
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

**Comment by rezarex** [comment]
Sorry @SurajPratap10 , I dismissed by mistake, but I have made the changes!

**Comment by SurajPratap10** [comment]
> Sorry @SurajPratap10 , I dismissed by mistake, but I have made the changes!

no worries

**Review by SurajPratap10 — APPROVED** [review]
LGTM 🎉

**Review by iamthecloverly — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**